### PR TITLE
docs(readme): Fix up grammar and consistency

### PR DESCRIPTION
**Pull request by sixhobbits**
Hi DragonFly! 

We're Ritza, a boutique technical writing agency that specialises in creating documentation, guides and blog posts for very technical audiences. We've worked with companies like Sourcegraph and Replit to create content that software developers love.

Part of our mission is improving the quality of all technical materials, so here are some improvements to your README.

Please let me know if we can help you with any other content!

### CODE REVIEW

**Comment by romange** [comment]
Thank you, but now i am not sure what to do with this PR. We cannot guarantee establishing business relationships because of the contribution.

**Comment by sixhobbits** [comment]
We don't need/expect anything in return for this PR and I fully agree to the CLA etc (I couldn't work out exactly if I have to sign it explicitly somewhere as the link just went to a text file, but let me know if you need anything else from us to accept this)

**Comment by romange** [comment]
just confirming the CLA is OK :)
I will review the PR during the next few days, thanks!

File: README.md [hunk 1]
```diff
@@ -72,120 +74,105 @@ Memcached exhibited lower latency for the read benchmark, but also lower through
...
 ## <a name="roadmap-status"><a/>Roadmap and status
 
-Currently, Dragonfly supports ~185 Redis commands and all memcache commands besides `cas`.
-We are almost on par with Redis 5 API. Our next milestone will be to stabilize basic
-functionality and implement the replication API.
-If you see that a command you need, is not implemented yet, please open an issue.
+Dragonfly currently supports ~185 Redis commands and all Memcache commands besides `cas`. Almost on par with the Redis 5 API, Dragonfly's next milestone will be to stabilize basic functionality and implement the replication API. If there is a command you need that is not implemented yet, please open an issue.
 
-For dragonfly-native replication, we are designing a distributed log format that will support order of magnitude higher speeds.
+For Dragonfly-native replication, we are designing a distributed log format that will support exponentially higher speeds.
```
Comment (romange) [inline README.md:128]: please keep "order of magnitude"

**Review by romange — APPROVED** [review]

**Pull request merged**